### DISCUSSION TITLE
Estrutura de Tecnologia

### DISCUSSION THREAD

**Discussion by gabubellon** [apyb/comunidade#8] [Geral]
Lançando alguns pontos;
Seria interessante a APyB centralizar uma estrutura (cloud ou não) e ferramentas para TODAs as comunidades, fazendo com que o custo ficasse bem mais diluído, exemplo, hoje ApyB paga o freshdesk.

Vejo os seguintes pontos interessantes;

* Uma estrutura de máquinas (AWS, Google, DigitalOcean, um server ??) onde 1. Poderiamos usar para levantar ferramentas e estruturas para uso em eventos locais e centrais; 2. Poderiamos criar rotinas que hoje são pagas em ferramentas de terceiros (envio de email, controle de arquivos, centralizaçao de chamados) 3. Poderia ser disponiblizado sob demanda para pessoas que precisem estudar/treinar, apresentar algo;4. Sites de comunidadea
* Serviços online que podem ser usados por várias comunidades, eg uma assinatura anual do Canvas para fazermos artes, uma do StreamYard para criar lives, e a APyB+Comunidade criaria um modo de gerir as mesmas, 
* Materias de publicidade e identidade visual:Poderiamos ter uma pessoa/agencia centralizada no contrato da APyB para atender demandar de criação de peças, logos, identidade visuais, que possa ser reaproveitadas por toda a comunidade. Imagina por exemplo que todo ano a gente tem um template PythonBrasil de identidade e todas as comunidades podem replicar ou a mesma forma e conteudo

Por mais que isso não seja dar dinheiro diretamente as comunidades isso aliva muita muito, principlamente as pequernas que tem que pensar em todo o fluxo
Assinar -> Pagar -> Nota -> Reembolso -> Receber

Com um fluxo de ferramentas pagas pela Apyb temos só um
Pede Acesso -> Acesso consedido por X tempo -> Acesso revogado

**Comment by luzfcb** [comment]
@gabubellon Eu acharia interessante que fosse feito um levantamento do custo atual mensal e médio de manter toda a infraestrutura atual que a APyB paga e o custo (sendo ele pago ou não, ou seja, custo $$ ou de tempo) para que pessoas X, Y, Z mantenham tudo funcionando e como esse custo afeta o caixa da associação. De posse dessas informações, nós poderemos tomar melhores decisões com base no que podemos efetivamente fazer com as tecnologias disponíveis e pessoas disponíveis para manter tudo.

**Reply by gabubellon** [reply]
Acho que podemos fazer um levantamento de custos PyBR2020 e 2021 e pedir para as comunidades locais fazerem o mesmo caso tenham usado algo e pedido ajuda a APyB.
Da org desse ano vou jogar no nosso Post Mortem e trazer para cá

**Reply by luzfcb** [reply]
Pergunta: Essa discussão é especifica sobre a infra da PyBR ou é sobre a infra da APyB inteira? (o que inclui a PyBR, mas inclui outras coisas também) 

Inicialmente eu pensei que fosse a segundo caso, ou seja, a infra da APyB inteira

**Reply by gabubellon** [reply]
Para mim APyB inteira, até porque o que acontece na PyBr se espalha por 1 ano para o resto do pais (e mundo ??)

**Reply by gabubellon** [reply]
Até porque, pensando em StreamYard por exemplo, uma mesma conta atende infinitos canais de youtube ou até só gravações sem transmissões 
Um mesmo sistema de chamados (que já existe) pode ajudar a centralizar duvidas de pessoas de todo lugar e ter resposta padrões para toda comunidade
Um servidor para  Django por exemplo pode atender várias comunidades para fazer cadastro, ou até estudos ou qualquer coisa de Django faça e ser reaproveitado
Eu posso fazer um modulo de vendas aqui em Joinville prra minha comunidade pra verder camisetas (e eu voi gerenciar) e você pode reaproveitar o mesmo modulo pra vender outra coisa ai na sua comunidade

**Comment by biancarosa** [comment]
Essa discussão me interessa. Eu tô fazendo por exemplo um teste com o pretalx em uma droplet da digital ocean. Se for interessante escalarmos pra uma iniciativa da apyb pra eventos menores, eu super topo contribuir.
